doc/doxygen/scripts/mod_header.pl.in

```diff
@@ -9,7 +9,7 @@ if (m'</head>')
     print '<link rel="SHORTCUT ICON" href="deal.ico"></link>', "\n";
     print '<script type="text/javascript" src="$relpath^custom.js"></script>', "\n";
     print '<meta name="author" content="The deal.II Authors <authors@dealii.org>"></meta>', "\n";
-    print '<meta name="copyright" content="Copyright (C) 1998 - ', $year, ' by the deal.II authors"></meta>', "\n";
+    print '<meta name="copyright" content="Copyright (C) 1998 - 2016 - ', $year, ' by the deal.II authors"></meta>', "\n";
```

Comment:
I did not notice this before but we should manually exclude this file since it has `$year`.